src/precompile.jl

```diff
@@ -1,7 +1,25 @@
 function _precompile_()
     ccall(:jl_generating_output, Cint, ()) == 1 || return nothing
-    precompile(Tuple{typeof(Parsers.parse), Type{Int64}, String})
-    precompile(Tuple{typeof(Parsers.parse), Type{Float64}, String})
-    precompile(Tuple{typeof(Parsers.parse), Type{Date}, String})
+
+    options = Parsers.Options()
+    pos = 1
+    val = "123"
+    len = length(val)
+    for T in (String, Int32, Int64, Float64, BigFloat, Dates.Date, Dates.DateTime, Bool)
+        for buf in (codeunits(val), Vector(codeunits(val)))
+            Parsers.xparse(T, buf, pos, len, options)
+            if !(T === String)
+                Parsers.xparse(T, buf, pos, len, options, T)
```

Comment:
Is there a benefit to calling this 6-arg method? doesn't the 5-arg `xparse` call above call this same 6-arg `xparse` internally?

Reply:
You're right it seems like there isn't. Removed.